shared-libs/cht-datasource/test/local/person.spec.ts

```diff
@@ -87,100 +87,76 @@ describe('local person', () => {
 
     describe('getWithLineage', () => {
       const identifier = { uuid: 'uuid' } as const;
-      let getLineageDocsByIdInner: SinonStub;
-      let getLineageDocsByIdOuter: SinonStub;
-      let getContactLineageInner: SinonStub;
-      let getContactLineageOuter: SinonStub;
+      let mockFetchHydratedDoc: SinonStub;
 
       beforeEach(() => {
-        getLineageDocsByIdInner = sinon.stub();
-        getLineageDocsByIdOuter = sinon
-          .stub(Lineage, 'getLineageDocsById')
-          .returns(getLineageDocsByIdInner);
-        getContactLineageInner = sinon.stub();
-        getContactLineageOuter = sinon
-          .stub(Lineage, 'getContactLineage')
-          .returns(getContactLineageInner);
+        mockFetchHydratedDoc = sinon.stub(Lineage, 'fetchHydratedDoc');
       });
 
       it('returns a person with lineage', async () => {
-        const person = { type: 'person', _id: 'uuid', _rev: 'rev' };
-        const place0 = { _id: 'place0', _rev: 'rev' };
-        const place1 = { _id: 'place1', _rev: 'rev' };
-        const place2 = { _id: 'place2', _rev: 'rev' };
-        const lineageDocs = [person, place0, place1, place2];
-        const lineagePlaces = [place0, place1, place2];
-        getLineageDocsByIdInner.resolves(lineageDocs);
+        const personWithLineage = { type: 'person', _id: 'uuid', _rev: 'rev' };
+        const mockFunction = sinon.stub().resolves(personWithLineage);
+        mockFetchHydratedDoc.returns(mockFunction);
         isPerson.returns(true);
         settingsGetAll.returns(settings);
-        const personWithLineage = { ...person, lineage: true };
-        const copiedPerson = { ...personWithLineage };
-        getContactLineageInner.returns(copiedPerson);
 
         const result = await Person.v1.getWithLineage(localContext)(identifier);
 
-        expect(result).to.equal(copiedPerson);
-        expect(getLineageDocsByIdInner.calledOnceWithExactly(identifier.uuid)).to.be.true;
-        expect(isPerson.calledOnceWithExactly(settings, person)).to.be.true;
+        expect(result).to.equal(personWithLineage);
+        expect(result).to.deep.equal(personWithLineage);
```

Comment:
```suggestion
```

suggestion: can remove this line because it is just a duplicate of the one above.